mmcv/runner/checkpoint.py

```diff
@@ -106,14 +106,43 @@ def load(module, prefix=''):
 
 
 def get_torchvision_models():
-    model_urls = dict()
-    for _, name, ispkg in pkgutil.walk_packages(torchvision.models.__path__):
-        if ispkg:
-            continue
-        _zoo = import_module(f'torchvision.models.{name}')
-        if hasattr(_zoo, 'model_urls'):
-            _urls = getattr(_zoo, 'model_urls')
-            model_urls.update(_urls)
+    if digit_version(torchvision.__version__) <= digit_version('0.12.0'):
+        model_urls = dict()
+        # When the version of torchvision is lower than 0.13, the model url is
+        # not declared in `torchvision.model.__init__.py`, so we need to
+        # iterate through `torchvision.models.__path__` to get the url for each
+        # model.
+        for _, name, ispkg in pkgutil.walk_packages(
+                torchvision.models.__path__):
+            if ispkg:
+                continue
+            _zoo = import_module(f'torchvision.models.{name}')
+            if hasattr(_zoo, 'model_urls'):
+                _urls = getattr(_zoo, 'model_urls')
+                model_urls.update(_urls)
+    else:
+        # Since torchvision bumps to 0.13, the weight loading logic, model keys
+        # and model urls have been changed. We will load the old version urls
+        # in mmcv/model_zoo/torchvision_0.12.json to prevent from BC
+        # Breaking. If your version of torchvision is higher than 0.13.0,
+        # new urls will be added to `model_urls` additionally. You can get the
+        # newest torchvision model by resnet50.imagenet1k_v1.
+        json_path = osp.join(mmcv.__path__[0],
+                             'model_zoo/torchvision_0.12.json')
+        model_urls = mmcv.load(json_path)
+        for cls_name, cls in torchvision.models.__dict__.items():
+            if (not cls_name.endswith('_Weights')
+                    or not hasattr(cls, 'DEFAULT')):
+                continue
+            # Since `cls.DEFAULT` can not be accessed by iterate cls, we set
+            # default urls explicitly.
+            cls_key = cls_name.replace('_Weights', '').lower()
```

Comment:
We also need to make [model_name](https://github.com/open-mmlab/mmcv/pull/1867/files#diff-4c27586e21983f57940038e850e5f123f88ae278b9b7a832db4a187cb7451260R428)  lower.

Reply:
`cls_name` includes `model_name`. It has been converted to lower.